### ISSUE TITLE
Include date in title tag of calendar events

### ISSUE THREAD

**Issue by mcking65** [w3c/w3c-website#411]
**Describe the issue**

When you open the page for an instance of a repeating event, the title of the page does not include the date. So, if you open two different instances, both browser tabs have the same title.

**To reproduce**

1. Go to calendar.
2. Open the page for the instance of a repeating event that occurs on DateX in a new browser tab
3. Open the page for the instance of the same repeating event that occurs on DateY.
4. The two browser tabs have identical titles even though they are for different event instances.

**Expected behavior**

The title tag for an event instance page should have form:
```
<title>
date | event-name | calendar | w3c
</title>
```

**Screenshots**
N/A

**Additional context**

This issue affects accessibility. It takes additional time for screen reader users to find the information on a page that uniquely identifies it.

**Comment by LJWatson** [comment]
Given the general propensity for people to have lots of tabs open simultaneously, I'm surprised this isn't a usability problem for everyone? @mcking65 you're right though, that the task of locating the date is noticeably more arduous for screen reader users than for others, so appending the meeting date to the title would definitely help.

**Comment by jean-gui** [comment]
> ```
> <title>
> date | event-name | calendar | w3c
> </title>
> ```

This can certainly be implemented. A few questions though:
1) What time zone should be used for the date, the one of the event, UTC, or the one from the user agent?
2) By "calendar", do you mean the string "calendar", or something related to a group calendar (for example "Verifiable Credentials Working Group Calendar")? The latter can't be done because a event can be tied to several groups.
3) Should we also include the status (draft, tentative, confirmed, canceled) . If yes, where?

**Comment by mcking65** [comment]
@jean-gui  asked:
> This can certainly be implemented. A few questions though:
> 
> 1. What time zone should be used for the date, the one of the event, UTC, or the one from the user agent?

The date in the title should match the date of the event that is shown to the user in the page content.

> 2. By "calendar", do you mean the string "calendar", or something related to a group calendar (for example "Verifiable Credentials Working Group Calendar")? The latter can't be done because a event can be tied to several groups.

The string "Calendar".

> 3. Should we also include the status (draft, tentative, confirmed, canceled) . If yes, where?

I don't think so as that is not a part of the path of the page in the IA of the W3C site. A primary purpose of the title is to inform the user which page is displayed in the tab of the browser. This is also especially helpful when reviewing browser history.

**Comment by jean-gui** [comment]
Thanks @mcking65 I'm going to work on implementing that

**Comment by mcking65** [comment]
@jean-gui, thank you thank you for this improvement! It is a huge help!
